pages/facts.html

```diff
@@ -4,9 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/facts.css">
+  <title>Fun Facts</title>
 </head>
 <body>
-  
+  <header class="header_container">
+    <nav>
+      <ul class="nav_container">
+        <h1>Fun Facts</h1>
+        <li>
+          <a href="../index.html">Home</a>
+        </li>
+        <li>
+          <a href="gallery.html">Memes</a>
+        </li>
+      </ul>
+    </nav>
+  <section>
```

Comment:
This is where the header tag should be closed.

```suggestion
    </nav>
  </header>
  <section>
```